prompb/remote.proto

```diff
@@ -40,5 +41,5 @@ message Query {
 
 message QueryResult {
   // Samples within a time series must be ordered by time.
-  repeated prometheus.TimeSeries timeseries = 1;
+  repeated prometheus.TimeSeries timeseries = 1 [(gogoproto.nullable) = false];
```

Comment:
I'd avoid making unnecessary changes for now.

Reply:
this is unnecessary because it just saves memory allocations? not required for changing how remote write receives samples I guess :+1:

Reply:
Only reason to say this is because its read path not write path; was this necessary because of changes on the write path sharing helping functions? If not, try and avoid it.